### ISSUE TITLE
changes in my-app.html breaks polymer-starter-kit (2.0-preview) in IE11

### ISSUE THREAD

**Issue by LawrenceMok** [PolymerElements/polymer-starter-kit#1020]
I've identified that after the addition (0b67b51367f3754b369d00141dc604f2752ce7ee) of this part of code in src/my-app.html, polymer-starter-kit (2.0-preview) failed to work in IE11 when run with polymer serve:
```
   constructor() {
        super();

        // Get root pattern for app-route, for more info about `rootPath` see:
        // https://www.polymer-project.org/2.0/docs/upgrade#urls-in-templates
        this.rootPattern = (new URL(this.rootPath)).pathname;
      }
```
To be specific, this line caused the problem and when commented out, it works:
`        this.rootPattern = (new URL(this.rootPath)).pathname;`

**Comment by jsilvermist** [comment]
This is caused because IE11 doesn't support using `new URL(...).pathname`, updated to use alternative method for IE11 in #1021. Thanks for pointing this out @LawrenceMok!

**Comment by abdonrd** [comment]
Fixed by #1021!
